src/components/LeaderboardPage/LeaderboardPage.constants.ts

```diff
@@ -0,0 +1 @@
+export const CURRENT_USER_ID = 'user-3';
```

Comment:
A default current user id is hardcoded in constants (CURRENT_USER_ID = 'user-3'). Hardcoded user identifiers in source code can lead to surprising behavior and make components less portable.

**Sugerencia:**
```typescript
Prefer deriving the current user id from an authentication context or environment. If a literal default is required for local dev/storybook/tests, document it clearly in the file and consider using an env var or a fixture instead.
```